rocky/Makefile

```diff
@@ -54,8 +54,9 @@ export_migrations:
 	python manage.py export_migrations accounts 0001
 
 languages:
-	# Extracts strings to .po files which should be translated
-	python manage.py makemessages -i "venv/*" -i "build/*" -i "octopoes/*" -i "node_modules/*" --verbosity 2 --add-location file -a ; \
+	# Extracts strings to `.pot` file which should be translated
+  # Note that the creation of `.po` files is delegated to another tool (Weblate)
+	python manage.py makemessages -i "venv/*" -i "build/*" -i "octopoes/*" -i "node_modules/*" --verbosity 2 --add-location file -a --keep-pot
```

Comment:
I think you can also run this only on `rocky/locale/` or run make messages from within that directory (using `manage.py` or `django-admin` command). This way you can get rid of all the exclude options

Reply:
Tested this, but it's a limitation of makemessages sadly :/